src/components/app.jsx

```diff
@@ -25,24 +25,20 @@ const App = React.createClass({
           <div className="container" id="content" style={{marginBottom: 100}}>
             <section className="jumbotron" style={{marginTop: "1em"}}>
               <div className="row">
-                <div className="col-lg-3">
-                  <p>
-                    <img
-                      alt="Logo OpenFisca"
-                      className="img-responsive"
-                      src={url.resolve(config.websiteUrl, "/hotlinks/logo-openfisca.svg")}
-                    />
-                  </p>
+                <div className="col-md-3">
+                  <img
+                    alt="Logo OpenFisca"
+                    src={url.resolve(config.websiteUrl, "/hotlinks/logo-openfisca.svg")}
+                    style={{maxWidth: 200}}
```

Comment:
Since the layout is mostly based on text, this [should not be set in `px` but in `em`](http://stackoverflow.com/questions/609517/why-em-instead-of-px), to adjust gracefully to different font sizes.